README.md

```diff
@@ -118,6 +122,14 @@ previously from [here](https://github.com/biomage-ltd/iac/tree/master/releases/s
 
 to remove your deployment.
 
+#### *removing staging resources*
+
+Isolated staging environments copies files and records to create a scoped staging environment. Sometimes, staging fails. To clean up the staged data manually, run 
+
+    python3 biomage unstage my-sandbox-id --resources-only
+
+to remove the staged resources.
```

Comment:
it will be useful to provide a list on what things need to be deleted manually in the event of a failure and where to look for them

Reply:
Added in commit 0c0c96a